lib/ansible/modules/packaging/os/urpmi.py

```diff
@@ -106,13 +106,13 @@ def query_package(module, name, root):
 def query_package_provides(module, name, root):
     # rpm -q returns 0 if the package is installed,
     # 1 if it is not installed
-    cmd = "rpm -q --provides %s %s" % (name, root_option(root))
+    cmd = "rpm -q --whatprovides %s %s" % (name, root_option(root))
     rc, stdout, stderr = module.run_command(cmd, check_rc=False)
     return rc == 0
 
 
 def update_package_db(module):
-    cmd = "urpmi.update -a -q"
+    cmd = "/usr/sbin/urpmi.update -a -q"
```

Comment:
Rather than hard coding the path, use `get_bin_path`.

Reply:
Thanks for the advice, will do

Reply:
done